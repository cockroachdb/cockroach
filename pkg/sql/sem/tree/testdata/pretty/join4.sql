select * from
   a join b using(c),
   a natural join b,
   a join b on (c)
where true
