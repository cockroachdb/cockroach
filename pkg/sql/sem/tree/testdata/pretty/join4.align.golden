1:
-
SELECT
	*
FROM
	a
	JOIN
		b
	USING (c),
	a
	NATURAL JOIN
		b,
	a
	JOIN
		b
	ON
		c
WHERE
	true

6:
------
SELECT
	*
FROM a
     JOIN
		b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN
		b
     ON
		c
WHERE
	true

8:
--------
SELECT *
FROM a
     JOIN
		b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN
		b
     ON
		c
WHERE
	true

9:
---------
SELECT *
FROM a
     JOIN
		b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN
		b
     ON c
WHERE
	true

10:
----------
SELECT *
FROM a
     JOIN
		b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN
		b
     ON c
WHERE true

11:
-----------
SELECT *
FROM a
     JOIN b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN b
     ON c
WHERE true

16:
----------------
SELECT *
FROM a
     JOIN b
     USING (c),
     a
     NATURAL JOIN
		b,
     a
     JOIN b ON c
WHERE true

20:
--------------------
SELECT *
FROM a
     JOIN b
     USING (c),
     a
     NATURAL JOIN b,
     a
     JOIN b ON c
WHERE true

22:
----------------------
SELECT *
FROM a
     JOIN b USING (c),
     a
     NATURAL JOIN b,
     a
     JOIN b ON c
WHERE true

56:
--------------------------------------------------------
SELECT *
FROM a JOIN b USING (c), a NATURAL JOIN b, a JOIN b ON c
WHERE true

76:
----------------------------------------------------------------------------
SELECT * FROM a JOIN b USING (c), a NATURAL JOIN b, a JOIN b ON c WHERE true
